UIKitTests/UIView/UIViewTests+subviews.swift

```diff
@@ -61,6 +61,20 @@ class UIViewSubviewTests: XCTestCase {
         XCTAssertEqual(view.layer.sublayers!, [subview1.layer, subview2.layer])
     }
 
+    func testInsertSubviewAtIndex() {
```

Comment:
@janek does this also run on the iOS test target?

Reply:
yes :)